tasks/pkg-redhat/install-apm-inject-latest.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Install the APM inject library
+  yum:
+    name: datadog-apm-inject
+    state: latest # noqa 403
+    update_cache: yes
+  become: true
```

Comment:
I don't think we should use `become: true`. We expect these things to be configured at playbook/host level.